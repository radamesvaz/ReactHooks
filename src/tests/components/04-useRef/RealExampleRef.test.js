import React from 'react';
import { shallow } from 'enzyme';
import { RealExampleRef } from '../../../components/04-useRef/RealExampleRef';

describe('Haciendo pruebas sobre el componente <RealExampleRef />', () => {
    
    const wrapper = shallow(<RealExampleRef />);
    
     test('debe mostrar el componente correctamente ', () => {
         

        expect(wrapper).toMatchSnapshot();
        expect( wrapper.find('MultipleCustomHooks').exists() ).toBe(false)
     })
     
     test('debe de mostrar el componente <MultipleCustomHooks /> ', () => {
         

        wrapper.find('button').simulate('click');

       expect( wrapper.find('MultipleCustomHooks').exists() ).toBe(true)

     })
     
})

