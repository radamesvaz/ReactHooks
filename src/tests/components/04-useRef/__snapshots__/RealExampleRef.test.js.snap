// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Haciendo pruebas sobre el componente <RealExampleRef /> debe mostrar el componente correctamente  1`] = `
<div>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-outline-primary mt-5"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
